StyleCop.Analyzers/StyleCop.Analyzers.Test/Helpers/DiagnosticVerifier.Helper.cs

```diff
@@ -209,6 +209,16 @@ protected DiagnosticResult CSharpDiagnostic(DiagnosticDescriptor descriptor)
             return new DiagnosticResult(descriptor);
         }
 
+        protected DiagnosticResult CSharpCompilerError(string errorIdentifier, string message)
+        {
+            return new DiagnosticResult
+            {
+                Id = errorIdentifier,
+                Severity = DiagnosticSeverity.Error,
+                Message = message,
```

Comment:
:thought_balloon: It would be more consistent with the other `DiagnosticResult` factory methods if the message was omitted and left for a separate call to `WithMessage` instead.